### PR TITLE
Update `dataloaders` params in example

### PR DESCRIPTION
**Pull request by kaushikb11**
## What does this PR do?

Also, no deprecation warning then.

## Before submitting
- [] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [] Did you make sure to update the documentation with your changes? (if necessary)
- [] Did you write any new necessary tests? (not for typos and docs)
- [] Did you verify new and existing tests pass locally with your changes?
- [] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pl_examples/bug_report_model.py [hunk 1]
```diff
@@ -53,13 +53,13 @@ def run():
...
-        limit_val_batches=1,
+        limit_train_batches=30,
+        limit_val_batches=30,
         num_sanity_val_steps=0,
-        max_epochs=1,
+        max_epochs=3,
         weights_summary=None,
     )
-    trainer.fit(model, train_dataloader=train_data, val_dataloaders=val_data)
+    trainer.fit(model, train_dataloaders=train_data, val_dataloaders=val_data)
```
Comment (awaelchli) [inline pl_examples/bug_report_model.py:62]: great!

File: pl_examples/bug_report_model.py [hunk 2]
```diff
@@ -53,13 +53,13 @@ def run():
     model = BoringModel()
     trainer = Trainer(
         default_root_dir=os.getcwd(),
-        limit_train_batches=1,
-        limit_val_batches=1,
+        limit_train_batches=30,
+        limit_val_batches=30,
         num_sanity_val_steps=0,
-        max_epochs=1,
+        max_epochs=3,
```
Comment (awaelchli) [inline pl_examples/bug_report_model.py:59]: can we keep these to 1? It's useful for debugging to just have one single step.

Comment (kaushikb11) [reply pl_examples/bug_report_model.py:59]: Is it tho? I always increase it when I use it for debugging ahha (also, the steps are tiny)

Comment (kaushikb11) [reply pl_examples/bug_report_model.py]: Updated!

**Review by awaelchli — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by carmocca — APPROVED** [review]
